endpoint.go

```diff
@@ -858,3 +861,25 @@ func (c *controller) cleanupLocalEndpoints() {
 		}
 	}
 }
+
+func (ep *endpoint) checkDefaultRoute() error {
```

Comment:
I don't think libnetwork should go in such detail and run this kind of checks.
A driver which passes both default gw and default static route seems non intentional, it should be a caught during testing time.